lib/kitchen/directions/bake_further_research.rb

```diff
@@ -24,6 +24,8 @@ def bake(chapter:, metadata_source:, uuid_prefix: '.')
 
           chapter.non_introduction_pages.each do |page|
             further_research = page.first('.further-research')
+            next unless further_research.present?
```

Comment:
I have added this line, to prevent the error I've got when I baked sociology using the general `bake` script  (not `shorten`): 
![image](https://user-images.githubusercontent.com/55203598/119036081-a2b3b100-b9b0-11eb-9d2c-70c5c276fe62.png)